samples/subsys/ipc/ipm_mcux/sample.yaml

```diff
@@ -3,7 +3,7 @@ sample:
         the lpcxpresso54114 using a mailbox.
     name: IPM MCUX Mailbox Sample
 tests:
-    sample.subsys.ipc.ipm_mcux:
+    sample.ipc.ipm_mcux:
```

Comment:
If you named test below `sample.ipc.ipm.ipm_mhu_dual_core:`, that test should be named like `sample.ipc.ipm.mcux:`